src/Http/Http.Features/src/IResponseBodyPipeFeature.cs

```diff
@@ -9,6 +9,7 @@ namespace Microsoft.AspNetCore.Http.Features
     /// <summary>
     ///     Represents the HttpResponseBody as a PipeWriter
     /// </summary>
+    [Obsolete("TODO: Remove, use IHttpResponseBodyFeature instead.", error: false)]
```

Comment:
I assume this is to keep the change small for now but you plan to remove it?

Reply:
Correct, incrementally marking things to remove once they're no longer used.